components/Paper/Tabs/PaperTab.js

```diff
@@ -154,7 +154,7 @@ function PaperTab(props) {
       );
     }
     if (
-      !paper.pdf_copyright_allows_display ||
+      !paper.pdf_copyright_allows_display &&
       !paper.pdfCopyrightAllowsDisplay
```

Comment:
I still want to check both fields for redundancy/robustness purposes, since this is pretty important logic.